bindata/etcd/pod.yaml

```
@@ -311,6 +311,28 @@ ${COMPUTED_ENV_VARS}
         name: log-dir
       - mountPath: /etc/kubernetes/static-pod-certs
         name: cert-dir
+  - name: etcd-backup-server
+    image: ${OPERATOR_IMAGE}
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: [cluster-etcd-operator]
```

this will loop the container because this prints the command help. 

```suggestion
    command: ["cluster-etcd-operator", "backup-server"]
```

I guess

---

yeah i wanted to do this since the very beginning, then the args are being added by the `BackupVars` completely 👍🏽

---

fixed